CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Fix error trace reporting for functions executed with `Server:exec()`
   (gh-396).
 - Remove pretty-printing of `luatest.log` arguments.
+- Fixed calculation of coverage percentage (gh-402)
```

Comment:
Bad changelog - unclear what exactly was wrong. Would be nice to say that it added uncovered files to the coverage report.

Also, please write a descriptive commit message. It should include `Closes #XXX`. We don't start subject lines with "fix: ...". Should probably be "Add uncovered files to coverage report".